hnn_core/viz.py

```diff
@@ -569,13 +569,87 @@ def plot_cell_morphology(cell, ax, show=True):
     return ax
 
 
+def plot_connectivity_weights(net, conn_idx, ax=None, show=True):
+    """ Plot connectivity matrix with color bar for synaptic weights
+    Parameters
+    ----------
+    net : Instance of Network object
+        The Network object
+    conn_idx : int
+        Index of _Connectivity object to be visualized
```

Comment:
could we avoid talking about `_Connectivity` since it's a private object? You can just say `net.connectivity`. It's just a dict for most users